services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -324,8 +385,8 @@ on a #switchType switch"() {
         Map<SwitchId, List<MeterEntry>> originalMeters = [src.dpId, dst.dpId].collectEntries {
             [(it): northbound.getAllMeters(it).meterEntries]
         }
-        def defaultMeters = { it.meterId < MAX_SYSTEM_RULE_METER_ID }
-        def flowMeters = { it.meterId >= MAX_SYSTEM_RULE_METER_ID }
+        def defaultMeters = { it.meterId <= MAX_SYSTEM_RULE_METER_ID }
+        def flowMeters = { it.meterId > MAX_SYSTEM_RULE_METER_ID }
```

Comment:
These can be spec-level closures so that you can reuse them in your test

Reply:
Yes, makes sense. Thanks.

Reply:
Fixed.